pkg/operator/k8sutil/labels.go

```diff
@@ -58,3 +60,13 @@ func AddRecommendedLabels(labels map[string]string, appName, parentName, resourc
 	labels["app.kubernetes.io/created-by"] = "rook-ceph-operator"
 	labels["rook.io/operator-namespace"] = os.Getenv(PodNamespaceEnvVar)
 }
+
+// LabelHostname returns label name to identify k8s node hostname
+func LabelHostname() string {
+	// todo: wait https://github.com/rook/rook/pull/15206
+	// return k8sutil.GetOperatorSetting("ROOK_CUSTOM_HOSTNAME_LABEL", corev1.LabelHostname)
+	if label, _ := os.LookupEnv("ROOK_CUSTOM_HOSTNAME_LABEL"); label != "" {
```

Comment:
placeholder logic. waits for #15206

Reply:
`Getenv()` should work fine, then no need to ignore the `ok` return value.
```suggestion
	if label := os.Getenv("ROOK_CUSTOM_HOSTNAME_LABEL"); label != "" {
```

Reply:
> placeholder logic. waits for #15206

Actually I don't think we need to wait for #15206. This PR could proceed using  the env var. Then when #15206 is merged, it would just add the capability to specify the setting in the operator configmap instead of only an env var.

Reply:
Geat, will it be possible to support it in helm chart?

I was thinking about adding `discover.customHostnameLabel` to [values.yaml](https://github.com/rook/rook/blob/a9af67d68532b5686c588f0f7c650e24df2432ac/deploy/charts/rook-ceph/values.yaml#L594) which will set envar to operator deployment template. Is it a good approach or it will be better to wait for configmap PR?

Reply:
`customHostnameLabel` makes sense, no need to put it under `discover`. The helm value can be implemented as an env var now, then we can switch the helm implementation to use the configmap after the other PR is merged. But the `customHostnameLabel` setting in the helm values can remain the same, so users won't even notice the change in the helm chart.